tests/test_mdib.py

```diff
@@ -59,6 +64,175 @@ def test_get_descriptor_by_code(self):
         self.assertIsNotNone(found2)
         self.assertEqual(handle, found2.Handle)
 
+    def test_copy_report_node(self):
+        def test_xml(raw_xml: bytes):
+            body = etree_.fromstring(raw_xml)[1]
+            for report in body:
+                new_report = xmlparsing.copy_node(report)
+                assert new_report != report  # check reference

```

Comment:
please use "self.assert..." instead of this kind of assert for such a unittest

also applies to two other lines below

Reply:
done